src/main/java/net/atos/zac/app/formulieren/converter/RESTFormulierVeldDefinitieConverter.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.formulieren.converter;
+
+import java.util.List;
+
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import net.atos.zac.app.formulieren.model.RESTFormulierVeldDefinitie;
+import net.atos.zac.formulieren.model.FormulierVeldDefinitie;
+
+public class RESTFormulierVeldDefinitieConverter {
+
+    public RESTFormulierVeldDefinitie convert(final FormulierVeldDefinitie veldDefinitie) {
+        final RESTFormulierVeldDefinitie restVeldDefinitie = new RESTFormulierVeldDefinitie();
+        restVeldDefinitie.id = veldDefinitie.getId();
+        restVeldDefinitie.systeemnaam = veldDefinitie.getSysteemnaam();
+        restVeldDefinitie.volgorde = veldDefinitie.getVolgorde();
+        restVeldDefinitie.label = veldDefinitie.getLabel();
+        restVeldDefinitie.type = veldDefinitie.getType();
+        restVeldDefinitie.beschrijving = veldDefinitie.getBeschrijving();
+        restVeldDefinitie.helptekst = veldDefinitie.getHelptekst();
+        restVeldDefinitie.defaultWaarde = veldDefinitie.getDefaultWaarde();
+        if (StringUtils.isNotBlank(veldDefinitie.getMeerkeuzeWaarden())) {
+            restVeldDefinitie.meerkeuzeWaarden = List.of(StringUtils.split(veldDefinitie.getMeerkeuzeWaarden(), "|"));
```

Comment:
Ik weet niet precies de betekenins van "|" maar wellicht is het een idee om hier een constante van te maken en deze te gebruiken op alle plaatsen waar hij nu staat.